options/locale/locale_en-US.ini

```diff
@@ -180,7 +180,7 @@ occurred = An error occurred
 report_message = If you believe that this is a Gitea bug, please search for issues on <a href="https://github.com/go-gitea/gitea/issues" target="_blank">GitHub</a> or open a new issue if necessary.
 missing_csrf = Bad Request: no CSRF token present
 invalid_csrf = Bad Request: invalid CSRF token
-not_found = The target couldn't be found.
+not_found = "The target couldn't be found."
```

Comment:
@wxiaoguang wasn't there something about quotes being bad for Crowdin?

Reply:
They're being used in many strings already.....

Reply:
The only problem is something like `key = "the quoted string" is at the beginning`

https://github.com/go-gitea/gitea/blob/main/modules/translation/i18n/i18n_test.go#L120C1-L120C93